djmoney/models/fields.py

```diff
@@ -34,12 +34,17 @@
 if 'djmoney_rates' in settings.INSTALLED_APPS:
     try:
         from djmoney_rates.utils import convert_money
```

Comment:
These things should be handled on a conventional level IMO.

A convention to *never* download anything in `djmoney/__init__.py` is very reasonable! We already have an issue open on this.

I've opened up #208 to explicitly deal with this.